.storybook/main.js

```diff
@@ -22,9 +23,16 @@ module.exports = {
           oneOf: [
             {
               test: /\.(png|jpe?g|gif)$/i,
+              use: [{loader: 'file-loader'}],
+            },
+            {
+              test: /\.(woff|woff2|eot|ttf)$/i,
```

Comment:
Да наверное можно затащить только woff2 шрифт, остальные для поддержки старых браузеров на клиентах нужны
```suggestion
              test: /\.(woff2)$/i,
```